src/encoded/static/scss/encoded/modules/_forms.scss

```diff
@@ -90,10 +96,12 @@ input[type="text"], input[type="password"], input[type="date"], input[type="date
 
 .search-wrapper {
 	position: 0;
```

Comment:
This line could be deleted as part of these changes. It's giving an "invalid property" error.

Reply:
@zoldello, @forresttanaka, and I discussed this, and the branch to get rid of bootstrap gets rid of this class entirely so this will be resolved when that branch is merged.